service/src/main/kotlin/no/nav/su/se/bakover/service/revurdering/RevurderingServiceImpl.kt

```diff
@@ -244,6 +249,41 @@ internal class RevurderingServiceImpl(
         }
     }
 
+    override fun forhåndsvarsleEllerSendTilAttestering(
+        revurderingId: UUID,
+        saksbehandler: NavIdentBruker.Saksbehandler,
+        revurderingshandling: Revurderingshandling,
+        fritekst: String,
+    ): Either<KunneIkkeForhåndsvarsle, Revurdering> {
+        val revurdering = revurderingRepo.hent(revurderingId)
+        if (revurdering?.forhåndsvarsel != null) {
+            return KunneIkkeForhåndsvarsle.AlleredeForhåndsvarslet.left()
+        }
+        return when (revurdering) {
+            is SimulertRevurdering -> {
```

Comment:
Henger det at vi bare forhåndsvarsler simulerte revurderinger sammen med hvordan vi oppfører oss ved f.eks  opphør? Regner med at dette vil kunne endre seg hvis/når vi åpner for revurdering av vilkår?

Reply:
Videre, finnes det noen andre tjenster som burde ta høyde for at vi kan være i "forhåndsvarsel-tilstand"?

Reply:
Kan se for meg at det kommer til å endre seg når vi åpner for vilkår. Per nå, så er det bare IngenEndring, og g-regulering(????), som ikke skal forhåndsvarsles